src/pip/_internal/locations.py

```diff
@@ -75,16 +75,24 @@ def get_src_prefix():
 except AttributeError:
     user_site = site.USER_SITE
 
+def _get_bin_user():
+    scheme = "{}_user".format(os.name)
+    if scheme not in sysconfig.get_scheme_names():
+        scheme = "posix_user"  # Default to POSIX for unknown platforms.
+    path = sysconfig.get_path("scripts", scheme=scheme)
+    assert path is not None
+    return path
+
+bin_user = _get_bin_user()
+
 if WINDOWS:
     bin_py = os.path.join(sys.prefix, 'Scripts')
-    bin_user = os.path.join(user_site, 'Scripts')
     # buildout uses 'bin' on Windows too?
     if not os.path.exists(bin_py):
         bin_py = os.path.join(sys.prefix, 'bin')
         bin_user = os.path.join(user_site, 'bin')
```

Comment:
Need to fix this `bin_user` as well (something like `os.path.join(os.path.dirname(bin_user), "bin")`)

Reply:
Sorry for the late response, I'm not sure what you mean here, you mean move this into `_get_bin_user`?

Reply:
This `bin_user` also needs to use `sysconfig` to get its config. Since this is a hack anyway, I would just do something like `bin_user = os.path.join(os.path.dirname(bin_user), 'bin')`